### PR TITLE
xinit: Move from -display argument to DISPLAY env variable

### PR DESCRIPTION
**Pull request by rminnich**
Too many programs have forgotten about -display.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: cmds/xinit/xinit.go [hunk 1]
```diff
@@ -32,10 +32,21 @@ func setup() error {
...
+			// We used to set -display :0, which all X programs
+			// are supposed to accept.
+			// wingo can not take the -display option at all.
+			// It gets an error.
+			// That option doesn't work with midori.
+			// It does not get an error, it just ignores the option.
+			// It's kind of weird how many of the basic X rules
+			// are no longer known.
+			cmd := exec.Command(n)
+			cmd.Env = append(os.Environ(), "DISPLAY=:0")
```
Comment (bkochendorfer) [inline cmds/xinit/xinit.go]: Do you want to remove the `-display` flag from the `aterm` command as well? It looks like absent the flag it should use the env variable. The outcome should be the same just wondering about consistency.

Comment (rminnich) [reply cmds/xinit/xinit.go]: let's look at this in a bit, X is so inconsistent anyway I'm not sure this matters. Putting -display in the args is marginally more convenient. The linuxcon show network is so bad I don't want to do this at present :-(

Comment (rminnich) [reply cmds/xinit/xinit.go]: well changing my mind, testing the change.

**Comment by rminnich** [comment]
This works, hope you like it :-)

**Pull request merged**